### ISSUE TITLE
pip install broken

### ISSUE THREAD

**Issue by chasehere** [Azure/azure-data-lake-store-python#136]
```
root@08e4d8f1cbb2:/# pip install azure-datalake-store
Collecting azure-datalake-store
  Downloading azure-datalake-store-0.0.2.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-fwmi3z5k/azure-datalake-store/setup.py", line 10, in <module>
        with open('HISTORY.rst', encoding='utf-8') as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'HISTORY.rst'
```

**Comment by paulpc** [comment]
bump

**Comment by begoldsm** [comment]
Thank you so much for the report @paulpc and @chasehere! This has been fixed in version 0.0.3, please install that version, as version 0.0.2 has an invalid setup.py.
